persistence.js

```diff
@@ -491,7 +513,7 @@ RedisPersistence.prototype.streamWill = function (brokers) {
 }
 
 RedisPersistence.prototype.getClientList = function (topic) {
-  var clientIds = this._matcher.match(topic).map(toClientIds)
+  var clientIds = this._matcher.match(topic).filter(isTopic).map(toClientIds)
```

Comment:
We had converted all maps, filters to straight for loops , not sure how we missed this